src/raylib.h

```diff
@@ -1334,6 +1334,7 @@ RLAPI Image ImageCopy(Image image);
 RLAPI Image ImageFromImage(Image image, Rectangle rec);                                                  // Create an image from another image piece
 RLAPI Image ImageText(const char *text, int fontSize, Color color);                                      // Create an image from text (default font)
 RLAPI Image ImageTextEx(Font font, const char *text, float fontSize, float spacing, Color tint);         // Create an image from text (custom sprite font)
+RLAPI Image ImageFromChannel(Image image, int selected_channel, float threshold);                        // Create an image from a selected channel of another image (Gray Alpha or RGBA)
```

Comment:
Please, could you review [raylib naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md), `selected_channel` should be `selectedChannel`.